resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -70,9 +71,15 @@ function task_service {
         else
             task install_source_rpm_systemd
         fi
-    elif [ "$OSFAMILY" = "Debian" ]; then
+      elif [ "$OSFAMILY" = "Debian" ]; then
+        sysv_codenames=("squeeze" "wheezy" "lucid" "precise" "trusty")
+        codename=`facter os.distro.codename`
```

Comment:
We aren't guaranteed to have facter at build time. Also, I believe this task never gets called by the packaging, but is just around for ezbake source installs.